main/urls.py

```diff
@@ -79,6 +79,7 @@
             react,
             name="application-history",
         ),
+        path("admissions/", react, name="admissions"),
```

Comment:
I don't think this is used

Reply:
oops! I had it under this url until your pr merged with the `/review` url